abis_mapping/types/spatial.py

```diff
@@ -126,15 +132,23 @@ def transformer_datum_uri(self) -> rdflib.URIRef | None:
         """Getter for the transformed datum URI.
 
         Returns:
-            rdflib.URIRef: Uri corresponding to transformer datum if known, else None.
+            rdflib.URIRef: Uri corresponding to transformer datum.
+
+        Raises:
+            GeometryError: If the project default CRS is not a part
+                of the GEODETIC_DATUM fixed vocab.
         """
         # Retrieve vocab class
-        if (vocab := utils.vocabs.get_vocab("GEODETIC_DATUM")) is not None:
-            # Init with dummy graph and return corresponding uri
-            return vocab(graph=rdflib.Graph()).get(settings.Settings().DEFAULT_TARGET_CRS)
+        vocab = utils.vocabs.get_vocab("GEODETIC_DATUM")
+        default_crs = settings.Settings().DEFAULT_TARGET_CRS
 
-        # If vocab doesn't exist
-        return None
+        try:
+            # Init with dummy graph and return corresponding uri
+            return vocab(graph=rdflib.Graph()).get(default_crs)
+        except utils.vocabs.VocabularyError as exc:
+            raise GeometryError(
```

Comment:
while you're here can you change `GeometryError` class to inherit from `Exception` not `BaseException`